bench-tps/src/main.rs

```diff
@@ -96,6 +99,16 @@ fn main() {
                 keypairs.push(Keypair::from_bytes(&bytes).unwrap());
                 last_balance = primordial_account.balance;
             });
+
+        if keypairs.len() != tx_count * 2 {
+            eprintln!(
+                "Expected {} accounts in {}, only received {} (--tx_count mismatch?)",
+                tx_count * 2,
+                client_ids_and_stake_file,
+                keypairs.len(),
+            );
+            exit(1);
```

Comment:
Should we bail or just warn? Because as @danpaul000 noted it does work in some cases even with a mismatch.

Reply:
I went for bail cause I didn't want to bother thinking about under what conditions it would succeed, maybe if the tx_count ls lower?  But in all cases this seems like a misuse by the user and it's better they know right away they're doing something not really supported

Reply:
Don't need to think about it, just 
```
if tx_count != expected_tx_count {
    warn!("Mismatched tx_count detected. User, you are on your own. \
             Hope you know what you're doing.");
}
```
But I can see how that might not be super visible in warning someone when dealing with a multi-machine network full of logs.

Reply:
Yeah I see your point.  Both of them :)
Imma leave it as is.  If somebody is bothered by the excessive hand-holding, they're always welcome to send another PR